logging/google/cloud/logging_v2/gapic/metrics_service_v2_client_config.py

```diff
@@ -0,0 +1,48 @@
+config = {
+    "interfaces": {
+        "google.logging.v2.MetricsServiceV2": {
+            "retry_codes": {
+                "idempotent": ["DEADLINE_EXCEEDED", "INTERNAL", "UNAVAILABLE"],
+                "non_idempotent": []
+            },
+            "retry_params": {
+                "default": {
+                    "initial_retry_delay_millis": 100,
+                    "retry_delay_multiplier": 1.2,
+                    "max_retry_delay_millis": 1000,
+                    "initial_rpc_timeout_millis": 20000,
+                    "rpc_timeout_multiplier": 1.5,
+                    "max_rpc_timeout_millis": 60000,
+                    "total_timeout_millis": 90000
+                }
+            },
+            "methods": {
+                "ListLogMetrics": {
+                    "timeout_millis": 60000,
+                    "retry_codes_name": "idempotent",
+                    "retry_params_name": "default"
+                },
+                "GetLogMetric": {
+                    "timeout_millis": 60000,
+                    "retry_codes_name": "idempotent",
+                    "retry_params_name": "default"
+                },
+                "CreateLogMetric": {
+                    "timeout_millis": 60000,
+                    "retry_codes_name": "non_idempotent",
+                    "retry_params_name": "default"
+                },
+                "UpdateLogMetric": {
+                    "timeout_millis": 60000,
+                    "retry_codes_name": "non_idempotent",
+                    "retry_params_name": "default"
+                },
+                "DeleteLogMetric": {
+                    "timeout_millis": 60000,
+                    "retry_codes_name": "idempotent",
+                    "retry_params_name": "default"
+                }
+            }
+        }
+    }
+}
```

Comment:
License header missing in `logging/google/cloud/logging_v2/proto/__init__.py`,

Reply:
Actually, it looks like the header is missing in lots of the generated files.

Reply:
Yeah, protoc does not generate license headers. We have an internal bug to track that (it is an open question whether they will do it). For now we are letting that go.